src/components/App.tsx

```diff
@@ -619,6 +631,13 @@ export class App extends React.Component<AppProps, AppState> {
               this.logLn("Template doesn't contain a project definition.", "error");
             } else {
               const json = await Service.loadProject(template.project, this.project);
+              if (template.onload) {
```

Comment:
See note below on handling templates. Also, calling the `onload` logic should also happen for fiddles that are loaded with an Id, not just created from a template.